js/ui/tag_box.js

```diff
@@ -680,6 +680,7 @@ const TagBox = SelectBox.inherit({
         if(isEmptyInput && canTypeText) {
             const $calculationElement = createTextElementHiddenCopy($input, value, { includePaddings: true });
 
+            $calculationElement.css('whiteSpace', 'pre');
```

Comment:
Are there any limitations to doing this by default for the `createTextElementHiddenCopy` utility?

Reply:
I fixed, but it's strange to me that `whiteSpace` was defined in element styles as `nowrap`. What is the likelihood that this definition just appeared there without any reason?